Tests/NIOEmbeddedTests/AsyncTestingEventLoopTests.swift

```diff
@@ -507,7 +507,9 @@ final class NIOAsyncTestingEventLoopTests: XCTestCase {
             try await group.waitForAll()
         }
 
-        XCTAssertGreaterThan(tasksRun.load(ordering: .acquiring), 1)
+        try await eventLoop.executeInContext {
```

Comment:
Does this actually run? Isn't the EL shutdown by now?

Reply:
When I stick a print in there, it shows. When I change the condition, it fails. So it looks like it runs.